1907-01-28.xml

```diff
@@ -2621,9 +2645,5503 @@
+                <div type="item">
+                    <head>Khedivial Mall Steamship and Graving Dock Company, Ltd.</head>
+                    <p>OFFICIAL NOTICE-PROPOSAL TO</p>
+                    <p>CHANGE A SHIP'S NAME.</p>
+                    <p>JOHN TEMPERLEY of 72, Bishopsgate-street</p>
+                    <p>Within, London, E.C., hereby give NOTICE that, in consequence of the recent
+                        transfer to The Khediviah Mail Steamship and Graving Dock Company, Limited,
+                        of the “Ifafe,” it is my intention to apply to the board of trade, under
+                        Section 47 of the Merchant Shipping Act, 1894, in respect of my ship "Ifaa,”
+                        of Aberdeen, Official Number 94,534. of gross tonnage 1,114.62 tons, of
+                        register tonnage 1,114.62 tons, heretofore owned by Darid Middleton Greig,
+                        of 25, Fay-sheet, Dundee ; Mary Greig or Keiller, of 13, Hyde-park gardens,
+                        London; Edward Pulleyo, of CliftonvilIe, York; Jane Maty Butcher, of Durban,
+                        Natal; Arthor George Butcher, of Darban, Natal; Mary Hustler Greenacre, of
+                        Durbar, Natal; Richmond George White, of Arcot, Sidmouth; James Jerram
+                        Pratt, of 79. Queen street, Cheapside; Margaret Irvine Wilson, of 2,
+                        Albyr-terrace, Aberdeen ; Sir James Stirling, High Court of Appeal, London;
+                        Hugh Stirling, 11, Birchin- lane, London ; James Hunter, Aberdeen ; George
+                        Hall Rennie, Aberdeen ; John Bylap, 4, East India-evenue, London ; Charles
+                        Reeves, of tyafban, Natal ; Alexander Wood, of Aberdeen; James Hatfter, of
+                        Temoral West Cults, Aberdeen ; John William, Burton, l3, Sherborne- lane,
+                        King William- street, London; Joseph Chiazzwi, of Durban, Natal; Harry James
+                        Butcher, of Darban, Natal; Leonard William Butcher, of Durban, Natal; Walter
+                        Edward Batcher, of Durban, Natal; Henry Howard Burton, of 13, Sherborne
+                        lane, King William-street, London; William Watson, of 182, Upper Tnames
+                        street, London ; Thomas Morgan Harvey, of Portland House, Basinghall street,
+                        London ; John Card-no Ogston Cooper, of Craigiehaokler, Aberdeen ; Caroline
+                        Harrod, of Calverwnod, Cross- in Hand, Surrey; Henry Herbert Harrod, of
+                        Wiverton Hotel Queen's gate, Kensington, London, Emily Mand Harrod, of 461,
+                        Barking rd., Canning Town,London; August A fholz, of 89, Highbury New Park,
+                        London ; John Thomas Heath, of 19. Charterhouse-street, London; George
+                        Rennie Stuart, of 4, East India-avenue, London; Agnes Wood, of West View,
```

Comment:
Cheap prepaid advertisements is a section (see template) that can contain all of its ads as items.